src/cclib/method/orbitals.py

```diff
@@ -22,12 +22,14 @@
 class Orbitals(Method):
     """A class for orbital related methods."""
 
-    def __init__(self, data, progress=None, \
+    def __init__(self, data, requiredAttr = ['mocoeffs','moenergies','homos'], progress=None, \
```

Comment:
By providing a default for `requiredAttr`, you're essentially implying that `requiredAttr` may contain more (or less!) than those three attributes, but it's up to the user of the `Orbitals` method to known which attributes are required.

Reply:
So the user would know of the required attributes, they just wouldn't know if those attributes were parsed into the ccData object?